src/components/DoctorCard/PageInfo.js

```diff
@@ -38,38 +124,65 @@ const PageInfo = function PageInfo({ doctor }) {
-          {doctor.fullAddress}
+          {isEditing ? (
+            <TextareaEdit name="inputAddress" value={inputAddress} setValue={setInputAddress} />
+          ) : (
+            doctor.fullAddress
+          )}
         </Typography>
-
         {urlText && (
           <Styled.PageInfo.LinkWrapper direction="row" alignItems="center" spacing={1}>
             <Typography component="div" variant="body1">
               <Icons.Icon name="Link" />
             </Typography>
-            <Shared.ConditionalLink to={doctor.website} variant="body1">
-              {urlText}
-            </Shared.ConditionalLink>
+            {isEditing ? (
+              <TextareaEdit name="inputWebsite" value={inputWebsite} setValue={setInputWebsite} />
+            ) : (
+              <Shared.ConditionalLink to={doctor.website} variant="body1">
+                {urlText}
+              </Shared.ConditionalLink>
+            )}
           </Styled.PageInfo.LinkWrapper>
         )}
-
         {doctor.phone && (
           <Styled.PageInfo.LinkWrapper direction="row" alignItems="center" spacing={1}>
             <Typography component="div" variant="body1">
               <Icons.Icon name="Phone" />
             </Typography>
-            <Shared.ConditionalLink to={doctor.phone && `tel:${doctor.phone}`} self variant="body1">
-              {doctor.phone}
-            </Shared.ConditionalLink>
+            {isEditing ? (
+              <TextareaEdit name="inputPhone" value={inputPhone} setValue={setInputPhone} />
+            ) : (
+              <Shared.ConditionalLink
+                to={doctor.phone && `tel:${doctor.phone}`}
+                self
+                variant="body1"
+              >
+                {doctor.phone}
+              </Shared.ConditionalLink>
+            )}
           </Styled.PageInfo.LinkWrapper>
         )}
-
         <Stack sx={{ mt: { md: 2 } }}>
           <Stack direction="row" alignItems="center" spacing={1}>
-            <Tooltip title={<Shared.Tooltip.HeadQuotient load={doctor.load} />}>
+            {isEditing ? (
               <Styled.InfoWrapper direction="row" alignItems="center" spacing={1}>
-                <Accepts accepts={accepts.toString()} />
+                <Accepts
+                  accepts={inputAvailability}
+                  setValue={setInputAvailability}
+                  isEditing={isEditing}
+                />
               </Styled.InfoWrapper>
-            </Tooltip>
+            ) : (
+              <Tooltip title={<Shared.Tooltip.HeadQuotient load={doctor.load} />}>
+                <Styled.InfoWrapper direction="row" alignItems="center" spacing={1}>
+                  <Accepts
+                    value={inputAvailability}
+                    setValue={setInputAvailability}
+                    isEditing={isEditing}
+                  />
+                </Styled.InfoWrapper>
+              </Tooltip>
+            )}
```

Comment:
I am not sure if should add new functionality to Accepts component at least not at this point. I would leave Accept component just to do one thing. Something just doesn't  feel right. Unless I am missing something.

I would go with some form component, which can be used somewhere else if needed and to be used something like `isEditing ? <SomeForm/> : <DoctorInfo/>`. Also it can be used in different ways (maybe as modal). And finally I believe it would be easier to implement different styles, especially having in mind that on mobile devices completing form can be difficult from user point of view.